src/spec/spec.cr

```diff
@@ -209,7 +209,7 @@ OptionParser.parse! do |opts|
     exit
   end
   opts.on("-v", "--verbose", "verbose output") do
-    Spec.formatter = Spec::VerboseFormatter.new
+    Spec.formatters << Spec::VerboseFormatter.new
```

Comment:
@asterite 

About the `VerboseFormatter`, from outside the module I'm able to push a new item to `Spec.formatters` but not able to replace it.

Like this:

```crystal
module NewSpec
  @@formatters = [] of String
  @@formatters << "wow"
  
  
  def self.formatters=(@@formatters)
  end
  
  def self.formatters
    @@formatters
  end
end

NewSpec.formatters << "foo"
NewSpec.formatters = ["foo", "bar"]

puts NewSpec.formatters.to_s
```
That works, but when trying to do this inside `src/spec/spec.cr:212`:

```crystal
Spec.formatters = [ Spec::VerboseFormatter.new ]
```
it raises: `error instantiating 'Spec:Module#formatters=(Array(Spec::VerboseFormatter))'`

Spec.formatters is an `Array(Spec::Formatter)` and `Spec::VerboseFormatter` is a `Spec::Formatter` so that should work, right?